ProjectManager/ProjectManager/List/ViewModel/ToDoListViewModel.swift

```diff
@@ -0,0 +1,143 @@
+//
+//  ToDoViewModel.swift
+//  ProjectManager
+//
+//  Created by Max on 2023/09/24.
+//
+
+import CoreData
+
+class ToDoListViewModel: ViewModelProtocol {
```

Comment:
```suggestion
final class ToDoListViewModel: ViewModelProtocol {
```

final, private같은 최적화는 습관입니다!

Reply:
넵! final 추가하도록 하겠습니다